tezos_interop/crypto.re

```diff
@@ -100,3 +100,101 @@ module Ed25519 = {
     let of_string = string => Base58.simple_decode(~prefix, ~of_raw, string);
   };
 };
+module Secp256k1 = {
+  open Libsecp256k1.External;
+  module K = Key;
+  let context = {
+    Mirage_crypto_rng_unix.initialize(); //TODO: adding this to to test files doesnt help ??
+    let c = Context.create();
+    let rand_value =
+      Mirage_crypto_rng.generate(32) |> Cstruct.to_bytes |> Bigstring.of_bytes;
+    let randomized = Context.randomize(c, rand_value);
+    randomized
+      ? c : failwith("Secp256k1 context randomization failed. Aborting.");
+  };
```

Comment:
Can you explain to me the job of this global context? And also, how is it done in Tezos?